docs/spec/api.md

```diff
@@ -2814,3 +3026,103 @@ The error codes that may be included in the response body are enumerated below:
 
 
 
+### Catalog
+
+List a set of available repositories in the local registry cluster. Does not provide any indication of what may be available upstream. Applications can only determine if a repository is available but not if it is not available.
+
+
+
+#### GET Catalog
+
+Retrieve a sorted, json list of repositories available in the registry.
```

Comment:
I'll add some language here to clarify this:

> The result set of repository names is represented abstractly as a lexically
sorted list, where the position in that list can be specified by the query
term `last`. The entries in the response start _after_ the term specified by
`last`, up to `n` entries.